File: prueba-de-envios-archivos-con-rama-samuel/practica.php
<?php

// Definimos un array con los números a verificar
$numeros = [20, 11, 19];

// Función para verificar si un número es primo
function esPrimo($numero) {
    if ($numero < 2) {
        return false;
    }
    for ($i = 2; $i <= sqrt($numero); $i++) {
        if ($numero % $i == 0) {
            return false;
        }
    }
    return true;
}

// Iteramos sobre el array de números
foreach ($numeros as $numero) {
    if (esPrimo($numero)) {
        echo "$numero es un número primo. ";
    } else {
        echo "$numero no es un número primo. ";
}
}
?>

File: prueba-de-envios-archivos-con-rama-samuel/practica.c
#include <stdio.h>

// Función para calcular el factorial
int calcularFactorial(int num)
{
    int factorial = 1;
    for (int i = 1; i <= num; ++i)
    {
        factorial *= i;
    }
    return factorial;
}

int main()
{
    int numero;


    // Solicitar al usuario que ingrese un número
    printf("Ingrese un número para calcular su factorial: ");
    scanf("%d", &numero);

    // Verificar que el número sea positivo
    if (numero < 0)
    {
        printf("El factorial no está definido para números negativos.\n");
    }
    else
    {
        // Calcular y mostrar el factorial del número ingresado
        int resultado = calcularFactorial(numero);
        printf("El factorial de %d es: %d\n", numero, resultado);
    }

    return 0;
}
